docs/user-guide/configuring-overview.md

```diff
@@ -0,0 +1,63 @@
+# Configuring Overview
+
+Review this article for an overview of the procedures that must be performed to configure Zowe z/OS components and the z/OS system. More details about the individual procedures are provided in the articles in this section. 
+
+:::info**Required roles:** system programmer, security administrator
+:::
+
+Configuring Zowe z/OS components consists of the following four main steps:
+
+1. Configure Zowe runtime
+2. Configure the z/OS system for Zowe
+3. Assign security permissions
+4. Configure the Zowe cross memory server (ZIS)
+
+:::note
+Successful completion of steps 2, 3, and 4 may require elevated security permissions. We recommend you consult with your security administrator to assist with performing these steps. 
+:::
+
+## Configuring Zowe runtime
+
+To cofigure Zowe runtime, choose from the following options:
+
+* Option 1: Configure Zowe manually using the `zwe init` command group
+* Option 2: Configure Zowe with z/OSMF workflows
+
+**Option 1**: To run the `zwe init` command, it is necessary to create a Zowe configuration file. For more information about this file, see [Zowe configuration file](./installandconfig/#zowe-configuration-file). 
+
+Once your configuration file is prepared, see [Initializing the z/OS system with `zwe init`](./initialize-zos-system.md), for more information about using the `zwe init` command group.
+
+**Option 2**: You can execute the Zowe configuration workflow either from a PSWI during deployment, or later from a created software instance in z/OSMF. Alternatively, you can execute the configuration workflow z/OSMF during the workflow registration process.
```

Comment:
I would merge these in just two bullet points 
Option 1: Configure Zowe manually using the `zwe init` command group 
  To run the `zwe init` command, it is necessary to create a <...> 
Option 2: Configure Zowe with z/OSMF workflows
  You can execute the Zowe configuration workflow <...>